src/css/profile/mobile/common/slider.less

```diff
@@ -38,14 +27,6 @@
 		position: absolute;
 		width: 100%;
 		display: flex;
```

Comment:
why remove positioning handler changes?

Reply:
Is "positioning handler" needed?
I can't find the value in the guide.
And when I remove it, the working is good.
There was no difference in the behavior with and without "positioning handler"

It it needed it?

Reply:
I don't know case when it would be needed. But if you remove it from LESS please remove it from JS fiel also.
src/js/core/widget/core/Slider.js (SPACE_BEFORE)

Reply:
Ok.
So It's prefer to add the "positioning handler".
I'll add it :)